.circleci/config.yml

```diff
@@ -649,6 +662,7 @@ workflows:
             - test-2.7
             - test-3.0
             - test-3.0-jit
+            - test-jruby-9.2.0
             - test-jruby-9.2
```

Comment:
Should we rename this one `jruby-9_2-latest` or something to better distinguish from the other?